src/chat/Chat.jsx

```diff
@@ -43,14 +58,35 @@ const getDirection = (userId, classes) => {
 const Chat = () => {
     const classes = useStyles();
 
+    const [messages, setMessages] = useState(dummyMessages);
+
+    const addDummyResponse = (newMessages) => {
+        const withResponse = newMessages.concat(getDummyResponse());
+        setMessages(withResponse);
+    }
+
+    const print = (text) => {
+        if(text.length > 0) {
+            const newMessages = messages.concat({userId: '1', text: text});
+            setMessages(newMessages);
+            setTimeout(function () {
+                addDummyResponse(newMessages);
+           }, 1000);
+        }
+    }
+
     const chatBubbles = messages.map((message, idx) => {
         return (<div className={`${classes.bubbleContainer} ${getDirection(message.userId, classes)}`} key={idx}>
             <div className={classes.bubble}>
                 <div className={classes.button}>{message.text}</div>
             </div>
         </div>);
     });
-    return <div className={classes.container}>{chatBubbles}</div>
+    return( 
+        <div className={classes.container}>
+            <div>{chatBubbles}</div> 
+            <MessageInput callBack={print} />
```

Comment:
![image](https://user-images.githubusercontent.com/5412431/77235029-362a0000-6bbb-11ea-9c56-a42586de5e32.png)

Add more spacing to the input field top, also make the field span the full width of the page

Reply:
I fixed this in the child components styles